tests/common/test_cgroupstelemetry.py

```diff
@@ -374,3 +374,44 @@ def test_extension_telemetry_not_sent_for_empty_perf_metrics(self, *args):  # py
                 metrics = CGroupsTelemetry.poll_all_tracked()
                 self.assertEqual(0, len(metrics))
 
+    @patch("azurelinuxagent.common.cgroup.MemoryCgroup.get_max_memory_usage")
```

Comment:
too many mocks here

Reply:
I think I followed old test cases. Actually, we don't need to mock memory usage as this test/change only for CPU values. I will remove them.

Reply:
ok, do you need the mock for is_active?